src/react/index.tsx

```diff
@@ -383,62 +384,61 @@ export const ConversationEmbed = componentFactory<
     ConversationViewConfig
 >(_ConversationEmbed);
 
-interface SpotterAgentEmbedProps extends EmbedProps, SpotterAgentEmbedViewConfig {}
+/**
+ * React component for individual conversation messages from SpotterAgent.
+ * This component is used internally by the useSpotterAgent hook.
+ * @version SDK: 1.37.0 | ThoughtSpot: 10.9.0.cl
+ */
+interface ConversationMessageProps extends EmbedProps, SpotterAgentMessageViewConfig {}
+
+export const ConversationMessage = componentFactory<
+    typeof _ConversationMessage,
+    ConversationMessageProps,
+    SpotterAgentMessageViewConfig
+>(_ConversationMessage);
+
+type SpotterMessageProps = {
+    message: SpotterAgentMessageViewConfig;
+    query?: string;
+  } & Omit<EmbedProps, keyof SpotterAgentMessageViewConfig>;
 
 /**
- * React component for SpotterAgent embed, which can be integrated inside
- * chatbots or other conversational interfaces.
+ * React component for displaying individual conversation messages from SpotterAgent.
+ * 
+ * This component renders a single message response from your ThoughtSpot conversation,
+ * showing charts, visualizations, or text responses based on the user's query.
+ * 
  * @example
  * ```tsx
- * function SpotterAgent() {
- *  const ref = useRef();
- *  
- *  const handleSendMessage = async () => {
- *    const { container, error } = await ref.current.sendMessage('show me sales by region');
- *    if (container) {
- *      document.body.appendChild(container);
- *    }
- *  };
- *  
- *  return (
- *    <div>
- *      <SpotterAgentEmbed ref={ref} worksheetId="worksheetId" />
- *      <button onClick={handleSendMessage}>Send Message</button>
- *    </div>
- *  );
+ * const { sendMessage } = useSpotterAgent({ worksheetId: 'worksheetId' });
```

Comment:
Can we update the developer example.

Reply:
https://github.com/thoughtspot/developer-examples/tree/main/visual-embed/spotter/spotter-agent-embed